apps/web/src/app/[locale]/(email)/newsletter/_components/newsletter-form.tsx

```diff
@@ -152,7 +152,7 @@ export function WaitlistForm() {
             text={
               // @ts-ignore
               mutation.error?.message === 'Member Exists'
-                ? 'You are already added to the waitlist with this email. We will reach out to you soon.'
+                ? 'You have already subscribed using this email!'
```

Comment:
correct, we dont return any emails just a string describing the error/scenario

Reply:
I know that you modified what was already there. 
If someone who doesn't own the email entered it into the input, he still would know that email is used by someone?

Reply:
Security. I know from my experience with app-sec guys. They always recommend displaying a message that you "successfully subscribed" once again, instead of that "email already is subscribed".

Reply:
Don't get me wrong. I don't have anything against with what you did 😄 . Just pointed a possible improvement, that I know security guys would point out

Reply:
I definitely see your point, I know it's usually a concern, but in our case, what is the security concern? I'm genuinely curious and I'm also indifferent to this! 

Let's say someone has your email already and they click on subscribe and they find out you're on TypeHero, what information did they gain? I can definitely make a new PR and modify this to return a success message on both success and error! cc: @bautistaaa